package.json

```diff
@@ -27,7 +27,7 @@
   },
   "peerDependencies": {
     "chai": "^3.0.0 || ^4.0.0",
-    "cheerio": "0.19.x || 0.20.x || 0.22.x || ^1.0.0-0",
+    "cheerio": "0.19.x || 0.20.x || 0.22.x || ^1.0.0-0 || ^1.0.0-rc.2",
```

Comment:
If you check https://semver.npmjs.com/, `^1.0.0-0` already matches `^1.0.0-rc.2`. There's no change needed here.

Reply:
yeah, i think i see the issue. yarn wants to see `cheerio` installed at the top level since `chai-enzyme` has it as a `peerDep`. without it installed there everything works but the warning shows up.

Reply:
Sounds like a yarn bug